var/spack/repos/builtin/packages/http-load/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class HttpLoad(MakefilePackage):
+    """http_load - multiprocessing http test client"""
+
+    homepage = "http://www.acme.com/software/http_load/"
+    url      = "http://www.acme.com/software/http_load/http_load-09Mar2016.tar.gz"
+
+    version('09Mar2016', sha256='a2bf118d88f6acd2a082cbf275d141538149caaa0a4ce1bcc66680668e781306')
```

Comment:
This will be problematic if anyone ever adds another version, as the version won't sort correctly. I would change the version of `2016-03-09` and either manually specify the URL for each version or use Python's datetime module to convert from one format to another in `url_for_version`. This is a suggestion, not a requirement, it could wait until someone actually needs another version.

Reply:
I tried this for your suggestion, is that OK?
If you approve this patch, I will apply it in the others package: `http-get`, `http-post`, `http-ping`

Reply:
Yes, this looks perfect! Please do the same thing for your other PRs. Although the month should be 3, not 5, correct?